docs/devel/networking.md

```diff
@@ -1,4 +1,14 @@
 # VMI Networking
+Before we explain how KubeVirt performs VM networking configuration, it is
```

Comment:
This may be just a matter of taste, but IMO it sounds better if we do not start with a semi-warning, but just guide the reader in a positive way. (not sure if I manage to express it right)
e.g the same can start as: `KubeVirt networking is layered over the k8s existing pod networking to support VM networking...`.
Feel free to ignore this and continue as is.